### ISSUE TITLE
i found one issue in new espeak which is available in nvda 2021.1 beta2 with persian language

### ISSUE THREAD

**Issue by zahra21** [espeak-ng/espeak-ng#964]
hi.
i always emphasized in several places that my favorite version of nvda is 2017.2 forever.
however, i sometimes wish to test newer versions to help developers and users.
today, i tested both 2021.1 beta one and 2021.1 beta two
i observed that in nvda speech settings, if i disable
Trust voice's language when processing characters and symbols
nvda does not announce persian kama at all.
،

**Comment by jaacoppi** [comment]
What is "persian kama"? Can you write an example sentence with expected output?

**Comment by Qchristensen** [comment]
I had a report from a user (on Twitter) about the following Persian pronunciation.  I'm not sure which version they were using:

They stated that دیگه=day gha'he, should be pronounced as دیگِ=dee ghe'h

**Comment by valdisvi** [comment]
In first comment mentioned character **،** is described as "Arabic comma".
I cannot reproduce reported behaviour for second comment.
```
espeak-ng -x -vfa دیگه
d'ige:
```
which en Englicised pronunciation would be `dee ghe'h` (so it looks correct).

**Comment by Qchristensen** [comment]
I've asked my user from Twitter to come and comment here, as I know absolutely no Persian and so can't help with what it should or shouldn't be reading.

**Comment by jaacoppi** [comment]
Closed until further information from original author is received.
